test/playlist.test.js

```diff
@@ -4,6 +4,64 @@ import QUnit from 'qunit';
 import xhrFactory from '../src/xhr';
 import { useFakeEnvironment } from './test-helpers';
 
+QUnit.module('Playlist Status');
```

Comment:
There is already an entire module of tests for blacklisting and enabled states near the bottom of this file...
https://github.com/videojs/videojs-contrib-hls/blob/master/test/playlist.test.js#L791